crates/sui-framework/packages/sui-framework/sources/authenticator_state.move

```diff
@@ -60,7 +60,8 @@ module sui::authenticator_state {
     #[allow(unused_function)]
     /// Create and share the AuthenticatorState object. This function is call exactly once, when
     /// the authenticator state object is first created.
-    fun create(ctx: &TxContext) {
+    /// Can only be called by genesis or change_epoch transactions.
+    public fun create(ctx: &TxContext) {
```

Comment:
Why does this need to be public?

Reply:
So that it can be called from sui_system - I tried to put this module in sui_system instead of sui and ran into some issues - i could try again.

Reply:
I don't think this can safely be public. Then anyone will be able to create the object

Reply:
Happy to help with whatever issues you ran into if it was in `sui_system` (which feels like a good home for this)

Reply:
@tnowacki it asserts that the sender is 0x0, so it should be safe. but i was able to remove the public anyway